core/src/test/java/io/grpc/ClientInterceptorsTest.java

```diff
@@ -351,6 +352,33 @@ protected void checkedStart(ClientCall.Listener<RespT> responseListener, Metadat
     ArgumentCaptor<Status> captor = ArgumentCaptor.forClass(Status.class);
     verify(listener).onClose(captor.capture(), any(Metadata.class));
     assertSame(error, captor.getValue().getCause());
+
+    // Make sure nothing bad happens after the exception.
+    ClientCall<?, ?> noop = ((CheckedForwardingClientCall<?, ?>)interceptedCall).delegate();
```

Comment:
Since the Exception was thrown in start() method, and after that the underlying call is NOOP_CALL, then why not just calling all of the methods there.